backend/protos/mgmt/v1alpha1/transformer.proto

```diff
@@ -232,6 +232,15 @@ message TransformPiiText {
 
   // Configure deny lists where each word is treated as PII.
   repeated PiiDenyRecognizer deny_recognizers = 3;
+
+  // Configure a list of recognizers to be used for PII analysis. If not provided or empty, all recognizers are used
+  // If this is specified, any ad-hoc, or deny_recognizers names must also be provided.
+  // To see available builtin recognizers, call the GetPiiTextRecognizers() RPC method to see what is available for your account.
+  repeated string allowed_recognizers = 4 [(buf.validate.field).repeated = {
```

Comment:
What do we think about this name?

Will it be confusing when looking at the property above called `deny_recognizers`?

Should we have a different name here?
wdyt: @alishakawaguchi @evisdrenova  ?

Maybe it makes more sense once we rename the `deny_recognizers` to just be `recognizers` or `adhoc_recognizers`

Reply:
Hmm I think it shoudl be called something different since it's not technically a recognizer as we have it defined in the `deny_recognizer` from a type perspective. I think it might be be clearer to call it something like `allowed_terms` or `allowed_words` or something along that line?

Reply:
sorry I was calling it the wrong thing as I was exposing the wrong function. this is will more clearly be named `allowed_entities` now